chore: mainnet deployment

XuJiandong opened the pull request:
Please review the deployment configuration by reading [this guide](https://github.com/XuJiandong/quantum-resistant-lock-script/blob/mainnet-deployment/deployment/mainnet/README.md) first.

After completing your review, please approve this deployment or leave a comment with "LGTM" if you don't have approval permissions. Once we receive approval from all 5 reviewers, the deployment will be signed and broadcast to the P2P network.

XuJiandong committed:
chore: mainnet deloyment information

XuJiandong committed:
docs: update README.md to clarify contact information

tea2x commented on deployment/mainnet/verify.sh:
```diff
@@ -0,0 +1,92 @@
+## step 1: verify binary
+function verify_binary() {
+    get_expected_hash() {
+        grep "build/release/c-sphincs-all-in-one-lock$" ../../checksums.txt | awk '{print $1}'
```
I got a question.

Shouldn't checksum be calculated on the binary instead of being grepped from checksum.txt?
Or maybe there should be a checksum validation prior to this function?

XuJiandong committed:
chore: fix reproducible build on mac(arm64)

On mac (arm64), Docker defaults to --platform=linux/arm64, while on Linux it defaults to --platform=linux/amd64. These are slightly different. Force Docker to use a single platform/image.

XuJiandong committed:
chore: previous UTXOs are spent

re-generate tx again

XuJiandong committed:
chore: locked by multisig v2

XuJiandong committed:
chore: update key of hanssen0@hanssen0.com

Hanssen0 reviewed (approved):
LGTM

XuJiandong committed:
chore: update pubkey from copperheart05@gmail.com

XuJiandong committed:
chore: deployed

XuJiandong committed:
docs: update README.md

XuJiandong commented:
deployed on mainnet:

| Parameter   | Value                                                                |
| ----------- | -------------------------------------------------------------------- |
| `code_hash` | `0x302d35982f865ebcbedb9a9360e40530ed32adb8e10b42fbbe70d8312ff7cedf` |
| `hash_type` | `type`                                                               |
| `tx_hash`   | `0x4598d00df2f3dc8bc40eee38689a539c94f6cc3720b7a2a6746736daa60f500a` |
| `index`     | `0x0`                                                                |
| `dep_type`  | `code`                                                               |
